IBPSA/Fluid/Storage/Stratified.mo

```diff
@@ -157,54 +157,6 @@ Icon(graphics={
           pattern=LinePattern.None,
           fillColor={0,0,127},
           fillPattern=FillPattern.Solid),
-        Rectangle(
-          extent={{-76,2},{-90,-2}},
-          lineColor={0,0,255},
-          pattern=LinePattern.None,
-          fillColor={0,0,127},
-          fillPattern=FillPattern.Solid),
-        Rectangle(
-          extent={{0,84},{-80,80}},
-          lineColor={0,0,255},
-          pattern=LinePattern.None,
-          fillColor={0,0,127},
```

Comment:
To be consistent with the other ports, why not put the middle one to the left?
![image](https://github.com/ibpsa/modelica-ibpsa/assets/22442514/053b904c-2175-48bf-a869-9528d4d5740c)

Reply:
I tested it but it looked weird in diagrams, especially when the middle port is left unconnected as in `IBPSA.Fluid.Storage.Examples.Stratified`.
However, I committed some modification to use `FluidPorts_a` instead of `FluidPort_a` to provide correct graphical feedback regarding the dimensionality of this port. 
I also relocated the port icon on the side of the tank.